### PR TITLE
add new company - 3Blocks

### PR DESCRIPTION
**Pull request by 01-pavan**
* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/3blocks.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Coupled with an advanced tech stack encompassing ReactJS, NestJS, Node.js, AWS, and Blockchain, our company leverages cutting-edge technologies to deliver innovative and scalable solutions, ensuring unparalleled value to our clients and users.
+
+## Office locations
+
+4910 E Adamo Dr, Unit B, Tampa, FL - 33605, United States.
+
+## How to apply
+
+Our open positions are listed on our [Linkedin](https://www.linkedin.com/company/3blocks-io/) page.
```
Comment (KieranRobson) [inline company-profiles/3blocks.md:29]:
```suggestion
Our open positions are listed on our [Linkedin](https://www.linkedin.com/company/3blocks-io/jobs/) page.
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change recommended. 

I see that there are no jobs listed, is there other evidence that this is a remote company?

**Comment by 01-pavan** [comment]
> Quick change recommended.
> 
> I see that there are no jobs listed, is there other evidence that this is a remote company?


@KieranRobson  you can see in https://www.linkedin.com/company/3blocks-io/people/ , the employees are located in various geographic locations.

File: company-profiles/3blocks.md (hunk 1 above)
Comment (01-pavan) [reply company-profiles/3blocks.md]: @KieranRobson updated the above LinkedIn jobs url

**Review by KieranRobson — APPROVED** [review]
LGTM

**Pull request merged**